apps/adguardhome/0.107.54/docker-compose.yml

```diff
@@ -22,7 +22,7 @@ services:
     volumes:
       - ./data/work:/opt/adguardhome/work
       - ./data/conf:/opt/adguardhome/conf
-    image: adguard/adguardhome:v0.107.54
+    image: adguard/adguardhome:v0.107.55
     labels:
       createdBy: "Apps"
 networks:
```

Comment:
The provided diff indicates that there is only one change in the Docker Compose configuration file: upgrading from AdGuardHome version `v0.107.54` to `v0.107.55`. This update ensures you're running a newer version of the software, which might include bug fixes or performance optimizations.

No other significant changes have been made, so no further action would be required unless these updates improve performance or compatibility with specific use cases.